desktop/extensions-sdk/guides/invoke-host-binaries.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Invoke host binaries
+description: Add invocations to host binaries from the frontend with the extension SDK.
+keywords: Docker, extensions, sdk, build
+---
+
+In some cases, your extension needs to invoke some command from the host (the computer of your users). For example, you 
+might wand to invoke the CLI of your cloud provider to create a new resource, or the CLI of a tool your extension 
+provides. 
+You could do that executing the CLI from a container with the extension SDK. But this CLI needs to access the host's 
+filesystem, which isn't possible if it runs in a container.
+Host binaries allows exactly this: to invoke binaries from the host. As extensions can run on multiple platforms, 
+this means that you need to ship the binaries for all the platforms you want to support.
+
```

Comment:
Add a "info" msg clarifying that you cannot run any binary on the host, but only those that are shipped as part of the extension.